.github/workflows/assets.yml

```diff
@@ -86,13 +110,25 @@ jobs:
           for asset in assets/*; do
             mv "$asset" "assets/${{ env.ARCH }}.$(basename "$asset")"
           done
-      - name: Upload release files
-        id: upload-release-files
-        uses: softprops/action-gh-release@v2
+          md5sum "assets/${{env.ARCH}}.rootfs.img" > assets/"${{env.ARCH}}.rootfs.img.md5"
```

Comment:
md5? why? sha256 would make more sense.

Reply:
Ah, yes we need sha not md5. Will fix it.

Reply:
this is fixed.